modules/apps/batch-engine/batch-engine-service/src/main/java/com/liferay/batch/engine/internal/reader/BatchEngineImportTaskItemReaderUtil.java

```diff
@@ -60,13 +62,29 @@ public static <T> T convertValue(
 				}
 			}
 
+			JsonDeserialize[] jsonDeserialize = field.getAnnotationsByType(
+				JsonDeserialize.class);
+
+			Object value = entry.getValue();
+
+			if ((jsonDeserialize.length > 0) &&
+				!Objects.equals(value.getClass(), String.class)) {
```

Comment:
I think this condition should not be included... I think the JsonDeserialize should work for any kind of field, it is already implemented in the JsonDeserializer...

Reply:
no longer applies.